src/styles/components/_card.scss

```diff
@@ -0,0 +1,95 @@
+.card {
+  $root: &;
```

Comment:
Do we really need this variable?

Reply:
why not? I don't see any reason for changing it